ast.go

```diff
@@ -54,6 +54,35 @@ func (ast *AST) PrettyPrint() {
 	spew.Dump("%#v", ast)
 }
 
+func SubtreeString (left string, operator string, right string) string {
```

Comment:
can make it private to the package by lowercasing the first letter: `subtreeString`